CONTRIBUTING.md

```diff
@@ -0,0 +1,93 @@
+# How To Contribute
+
+There are several ways to contribute to the development of ReactOS. The most often encountered problem is not knowing where to begin or what to do. If you are able to program or understand the technical information that is pertinent to this project, helping the development can be easy.
+
+- [What To Do?](#what-to-do?)
+- [How To Contribute?](#how-to-contribute?)
+- [Where To Start?](#where-to-start?)
+
+## What To Do?
+
+### Fix bugs
+
+You can try to fix a few bugs that are already listed in [JIRA]. Squashing bugs is not a simple task. It requires a lot more skill than simply searching for them, and can be time consuming; however, by doing that you greatly help ReactOS become a stable system.
+
+_NOTE: patches related to 3rd party code such as Wine or BtrFS should be sent to upstream of the said projects. See [3rd Party Files.txt], [README.WINE] and [README.FSD] in [media/doc](media/doc) for details._
+
+### Fix tests
+
+Tests are used to check the functionality and correctness of APIs on ReactOS compared to Windows implementations. There are some unit tests that you could help ReactOS pass, which can be found [in the Web Test Manager][testman] and some that are broken or yet to be written.
+
+### Fixing Coverity scans
```

Comment:
This should be "Fix" for consistency with the rest yeah?

Reply:
Yep, missed that.